concepticondata/conceptlists.tsv

```diff
@@ -359,9 +359,10 @@ Anonby-2018-1500	Anonby, Erik and Asadi, Ashraf and Nikravan, Pegah	2018		1500	a
 Majid-2015-21	Majid, Asifa and van Staden, Miriam	2015		21	body parts	English	Dutch, Japanese, Indonesian	https://doi.org/10.1111/tops.12159	Majid2015		This is the list of body part terms used in a coloring-in experiment by [Majid and van Staden (2015)](:bib:Majid2015). In some cases the closest translation equivalent across languages was not clear a priori so multiple terms were included (e.g., for belly). Glosses indicate current best translation. Participants were given a sheet of paper with a blank line drawing of an androgonized (originally female) body and had to color-in the given body part. The sheets were scanned afterwards and the colored-in areas were compared.		
 Blomberg-2020-160	Blomberg, Frida and Roll, Mikael and Frid, Johan and Lindgren, Magnus and Horne, Merle	2020		160	ratings	English	Swedish	https://www.jbe-platform.com/content/journals/10.1075/ml.19021.blo	Blomberg2020		This is a list of experimental stimuli (nouns only). The words were presented visually and the participants performed a lexical decision task while their EEGs were recorded. The words were grouped into four semantic categories: specific, general, emotional, abstract.		
 Vergallito-2020-1121	Vergallito, Alessandra and Petilli, Marco Alessandro and Marelli, Marco	2020		1121	ratings	English	Italian	https://osf.io/zdg59/	Vergallito2020		This is a list of perceptual modality norms for 1,121 Italian words (20% of adjectives, 69% of nouns, 5% of verbs and a 6% of words that could be considered both as an adjective or a noun.). Participants provided ratings on five modalities (visual, haptic, auditory, olfactory, or gustatory) on a scale of 0 (not at all) to 5 (greatly).		
+Jackson-2019-24	Jackson, Joshua Conrad and Watts, Joseph and Henry, Teague R. and List, Johann-Mattis and Mucha, Peter J. and Forkel, Robert and Greenhill, Simon J. and Lindquist, Kristen	2019		24	ratings, specific	English	Global	https://science.sciencemag.org/content/366/6472/1517	Jackson2019		This 24-item list underlies the study on the cultural variation and structure in emotion semantics across nearly 2500 languages from 20 major families. The data for this investigation was derived from the [database of cross-linguistic colexifications (CLICS)](http://clics.lingpy.org). The columns VALENCE, ACTIVATION, DOMINANCE, CERTAINTY, APPROACH_AVOIDANCE and SOCIALITY correspond with the respective physiological dimensions of the emotive concepts that were rated in a questionnaire. The ratings are available for 20 out of the 24 terms since the study was carried out before the data were extended.		
```

Comment:
Link for CLICS is clics.clld.org, clics.lingpy.org is the old version. emotive concepts -> emotion concepts. 

Reg. wording: "emotion concepts that were rated by users"